client/my-sites/email/email-management/home/email-plan-mailboxes-list.jsx

```diff
@@ -32,15 +36,40 @@ const getListHeaderTextForAccountType = ( accountType, translate ) => {
 	} );
 };
 
-const MailboxListHeader = ( { accountType = null, children, isPlaceholder = false } ) => {
+const MailboxListHeader = ( {
+	accountType = null,
+	children,
+	isPlaceholder = false,
+	addMailboxPath,
+	showIcon,
+} ) => {
 	const translate = useTranslate();
+	const isGoogle = isGoogleEmailAccount( { accountType } );
+
+	const HeaderIcon = () => (
+		<div className="email-plan-mailboxes-list__mailbox-header-icon">
+			{ isGoogle && <GoogleIcon /> }
+			{ ! isGoogle && <Icon icon={ wordpress } /> }
```

Comment:
I just realized we have an existing `EmailTypeIcon` component that could conceivably be a good fit here

Reply:
Nice catch, thanks!!! I replaced the custom logic with the existing `EmailTypeIcon` component.